### ISSUE TITLE
I got negative training loss and val loss when traing the example 04_Hippocampus. I'm confused why the loss is not positive.

### ISSUE THREAD

**Issue by Truring** [MIC-DKFZ/nnUNet#666]
![image](https://user-images.githubusercontent.com/22017747/123358372-f613b300-d59d-11eb-904a-740298b2f363.png)

**Comment by FabianIsensee** [comment]
why should the loss be positive? There is no reason it should be.
The Dice loss is -sift_dice and its best value is -1. The CE loss has best value 0, so -1 is a perfect model

**Comment by Chop1** [comment]
I would say most people shift the value of the loss function by doing somethink like ```return 1 - sift_dice```. I was also confused when I first saw the loss of nnUnet being negative.

**Comment by FabianIsensee** [comment]
objectively there is no reason to do this though :-D
